x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/searchable_snapshot_field/searchable_snapshot_field.tsx

```diff
@@ -267,7 +270,22 @@ export const SearchableSnapshotField: FunctionComponent<Props> = ({
             'xpack.indexLifecycleMgmt.editPolicy.searchableSnapshotCalloutBody',
             {
               defaultMessage:
-                'Force merge, shrink and freeze actions are not allowed when searchable snapshots are enabled in this phase.',
+                'Force merge, freeze, shrink and rollup actions are not allowed when searchable snapshots are enabled in this phase.',
+            }
+          )}
+          data-test-subj="searchableSnapshotFieldsDisabledCallout"
+        />
+      );
+    } else if (phase === 'cold' && isUsingSearchableSnapshotInColdPhase) {
+      infoCallout = (
+        <EuiCallOut
+          size="s"
+          title={i18n.translate(
+            'xpack.indexLifecycleMgmt.editPolicy.cold.searchableSnapshotCalloutBody',
+            {
+              defaultMessage:
```

Comment:
This is a good point! I put it under the searchable snapshot because in the hot phase we follow a similar pattern of showing the callout when you turn searchable snapshot on. The reasoning is that searchable snapshot is the thing that controls whether settings you may have set are available or not.

I think your point regarding actions being in advanced settings is valid, though! Perhaps we can spend some time coming up with a stronger pattern to follow here, but we should update this for the existing logic for hot phase too.

Reply:
CC @mdefazio